File: Assignment32/program4.java
import java.lang.*;
import java.util.*;

class Bitwise
{
    public int ToggleBit(int iNo, int iPos)
    {
        if((iPos<=0)||(iPos>32))
        {
            System.out.println("Invalid Position");
        }

         int iMask = 0X00000001;
         iMask = iMask<<(iPos-1);
         int iResult = iMask ^ iNo;
         
    return iResult;
    }
}


public class program4 {

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number");
        int value = sobj.nextInt();

        System.out.println("Enter the position");
        int pos = sobj.nextInt();

        Bitwise obj = new Bitwise();
        int iret = obj.ToggleBit(value, pos);

        
        System.out.println("updated number "+iret);
    }
    
}

File: Assignment39/program4.c
    #include<stdio.h>
int Min(int ino)
{

        int temp1=0,min=0;
    while(ino>0)
    {   
        
        temp1= ino%10;
        if(temp1<min)
        {
        min= temp1;
        }
        ino=ino/10;
    }
    return min;
}

int MinR(int ino)
{

    static int temp1=0,min=0;
    if (ino>0)
    {   
        
        temp1= ino%10;
        if(temp1>min)
        {
        min= temp1;
        }
        ino=ino/10;
        MinR(ino);
    }
    return min;
}
int main()
{
    int iRet=0;
    int value=0;
    printf("Enter number: \n");
    scanf("%d",&value);
    iRet = MaxR(value);
    printf("%d",iRet);
}

